LocalRunGuide.md

```diff
@@ -115,33 +135,40 @@ chmod +x ./gradlew &&\
 ./gradlew build
 ```
 
-
 ### Step 5: Move jar to desired location
 
 After the build process, a `.jar` file will be generated in the `build/libs` directory.
 You can move this file to a desired location, for example, `/opt/Stirling-PDF/`.
 You must also move the Script folder within the Stirling-PDF repo that you have downloaded to this directory.
-This folder is required for the python scripts using OpenCV
+This folder is required for the python scripts using OpenCV.
 
 ```bash
 sudo mkdir /opt/Stirling-PDF &&\
 sudo mv ./build/libs/Stirling-PDF-*.jar /opt/Stirling-PDF/ &&\
 sudo mv scripts /opt/Stirling-PDF/ &&\
 echo "Scripts installed."
 ```
+
+For non-root users, you can just keep the jar in the main directory of Stirling-PDF using the following command:
+```bash
+mv ./build/libs/Stirling-PDF-*.jar ./Stirling-PDF-*.jar
+```
+
 ### Step 6: Other files
 #### OCR
 If you plan to use the OCR (Optical Character Recognition) functionality, you might need to install language packs for Tesseract if running non-english scanning.
 
 ##### Installing Language Packs
-Easiest is to use the langpacks provided by your repositories. Skip the other steps
+Easiest is to use the langpacks provided by your repositories. Skip the other steps.
```

Comment:
Dont understand the wording here

Do you mean using a package manager is better than manual? 
Shall we change the order to move manual to bottom and package managers to top and remove this text?

Not sure what you mean by **Note:** Nix Package Manager pre-installs almost all the packages by default.

Reply:
Ah my bad, I meant pre-installs all the 'language' packages by default. Will fix this one up.